### PR TITLE
List search pages code, style, layout fixes.

### PR DESCRIPTION
**Pull request by ymarcon**

### CODE REVIEW

File: src/lists/lists-controller.js [hunk 1]
```diff
@@ -101,58 +78,21 @@ angular.module('obiba.mica.lists')
...
-        });
-
-        $scope.selected.sort = selectedSort;
-        emitter.$emit('ngObibaMicaSearch.sortChange', sortParam);
+      $scope.selectSortOrderOption = function (option) {
+        $scope.selected = option;
+        emitter.$emit('ngObibaMicaSearch.sortChange', option.value);
       };
 
       function intiSelectedOptions() {
```
Comment (kazoompa) [inline src/lists/lists-controller.js]: may be we could fix the spelling too.

Comment (ymarcon) [reply src/lists/lists-controller.js]: the spelling?

Comment (kazoompa) [reply src/lists/lists-controller.js]: intiSelectedOptions();

**Comment by kazoompa** [comment]
there are conflicts to resolve, once done I can merge the PR

**Comment by smirfolio** [comment]
We have to let some arrangement layout, because of an already validated pattern from Science Team:
individualstudies

**Comment by ymarcon** [comment]
@smirfolio this layout does not work because when there is no paging, it makes the sort widget alone on its row. That's why I changed for having both the count and the sorter on the same row. I don't think it is a good idea to keep the "approved" layout.

File: src/lists/views/search-result-list-template.html [hunk 2]
```diff
@@ -9,37 +9,39 @@
...
-<div ng-show="display === 'list'">
-    <div class="row voffset3">
-        <div class="col-md-12">
-            <span role="presentation" ng-repeat="res in resultTabsOrder"
-                  ng-class="{active: activeTarget[targetTypeMap[res]].active && resultTabsOrder.length > 1, disabled: resultTabsOrder.length === 1}"
-                  ng-if="options[targetTypeMap[res]].showSearchTab">
+    <div class="row">
+        <div class="col-md-2"><span role="presentation" ng-repeat="res in resultTabsOrder"
+                                     ng-class="{active: activeTarget[targetTypeMap[res]].active && resultTabsOrder.length > 1, disabled: resultTabsOrder.length === 1}"
+                                     ng-if="options[targetTypeMap[res]].showSearchTab">
```
Comment (ymarcon) [inline src/lists/views/search-result-list-template.html]: I totally disapprove these changes. Result is not good at all for pages other than the studies one.

File: src/lists/views/search-result-list-template.html [hunk 3]
```diff
@@ -8,71 +8,43 @@
...
-            ng-if="options[targetTypeMap[res]].showSearchTab">
-        <span ng-if="resultTabsOrder.length === 1">
-          <div>
-            {{totalHits = getTotalHits(res);""}}
-            {{singleLabel = "search." + res + ".label";""}}
-            <span>
-                {{totalHits | localizedNumber }}  {{totalHits>1?targetTypeMap[res]:singleLabel | translate}}
+<div ng-show="options.obibaListOptions.searchForm">
+    <div class="row">
+        <div class="col-md-2"><span role="presentation" ng-repeat="res in resultTabsOrder"
```
Comment (ymarcon) [inline src/lists/views/search-result-list-template.html]: Not a good idea to have it here. See my other comments.

**Comment by smirfolio** [comment]
networksdefaulttemplate
networks
individualstudiesdefaulttemplate
individualstudies

**Comment by ymarcon** [comment]
All in one line, overridable:
mica-studies
mica-networks2

**Pull request merged**